src/main/java/com/ericsson/ei/subscription/InformSubscriber.java

```diff
@@ -124,7 +124,7 @@ public void informSubscriber(String aggregatedObject, JsonNode subscriptionJson)
 
             if (!success) {
                 String missedNotification = prepareMissedNotification(aggregatedObject, subscriptionName, notificationMeta);
-                LOGGER.debug("Prepared 'missed notification' document : " + missedNotification);
+                LOGGER.debug("Prepared 'missed notification' document : {}", missedNotification);
```

Comment:
Please revert changes in this file, it will be refactored and a later pr.

Reply:
I'll just fix the conflict with your PR.